### PR TITLE
DBZ-2753: Modified instructions to use Podman to run the core example

### PR DESCRIPTION
**Pull request by rareddy**
Signed-off-by: Ramesh Reddy <rareddy@jboss.org>

### CODE REVIEW

**Comment by rareddy** [comment]
@jpechane @rk3rn3r for your review.

**Comment by jpechane** [comment]
@rareddy LGTM, but I wonder if it really needs to be tagged for community only version
@roldanbob Any thoughts on it?

**Comment by roldanbob** [comment]
@jpechane @rareddy   I'll take a look at it today.

File: documentation/modules/ROOT/partials/modules/tutorial/proc-cleaning-up.adoc [hunk 1]
```diff
@@ -22,6 +22,18 @@ Docker stops each container.
 Because you used the `--rm` option when you started them, Docker also removes them.
 --
 
+ifdef::community[]
+[NOTE]
+====
+For Podman use the below command
```
Comment (roldanbob) [inline documentation/modules/ROOT/partials/modules/tutorial/proc-cleaning-up.adoc:28]: If you use Podman, run the following command:

File: documentation/modules/ROOT/partials/modules/tutorial/proc-registering-connector-monitor-inventory-database.adoc [hunk 2]
```diff
@@ -104,6 +104,17 @@ Otherwise, you might see an error like the following:
 endif::[]
 --
 
+ifdef::community[]
+[NOTE]
+====
+For Podman use the below command
```
Comment (roldanbob) [inline documentation/modules/ROOT/partials/modules/tutorial/proc-registering-connector-monitor-inventory-database.adoc:110]: If you use Podman, run the following command:

File: documentation/modules/ROOT/partials/modules/tutorial/proc-starting-kafka-connect.adoc [hunk 3]
```diff
@@ -32,6 +32,17 @@ This enables applications outside of the container to use Kafka Connect's REST A
 `--link zookeeper:zookeeper --link kafka:kafka --link mysql:mysql`:: Links the container to the containers that are running ZooKeeper, Kafka, and the MySQL server.
 --
 
+ifdef::community[]
+[NOTE]
+====
+For Podman use the below command
```
Comment (roldanbob) [inline documentation/modules/ROOT/partials/modules/tutorial/proc-starting-kafka-connect.adoc:38]: If you use Podman, run the following command:

File: documentation/modules/ROOT/partials/modules/tutorial/proc-starting-kafka.adoc [hunk 4]
```diff
@@ -45,6 +45,17 @@ which means the terminal's standard input and output are attached to the contain
 `--link zookeeper:zookeeper`:: Tells the container that it can find ZooKeeper in the `zookeeper` container,
 which is running on the same Docker host.
 
+ifdef::community[]
+[NOTE]
+====
+For Podman use the below command
```
Comment (roldanbob) [inline documentation/modules/ROOT/partials/modules/tutorial/proc-starting-kafka.adoc:51]: If you use Podman, run the following command:

File: documentation/modules/ROOT/partials/modules/tutorial/proc-starting-mysql-command-line-client.adoc [hunk 5]
```diff
@@ -28,6 +28,17 @@ which means the terminal's standard input and output are attached to the contain
 `--link mysql`:: Links the container to the `mysql` container.
 --
 
+ifdef::community[]
+[NOTE]
+====
+For Podman use the below command
```
Comment (roldanbob) [inline documentation/modules/ROOT/partials/modules/tutorial/proc-starting-mysql-command-line-client.adoc:34]: If you use Podman, run the following command:

File: documentation/modules/ROOT/partials/modules/tutorial/proc-starting-mysql-database.adoc [hunk 6]
```diff
@@ -31,6 +31,17 @@ which means the terminal's standard input and output are attached to the contain
 `-e MYSQL_ROOT_PASSWORD=debezium -e MYSQL_USER=mysqluser -e MYSQL_PASSWORD=mysqlpw`:: Creates a user and password that has the minimum privileges required by the {prodname} MySQL connector.
 --
 
+ifdef::community[]
+[NOTE]
+====
+For Podman use the below command
```
Comment (roldanbob) [inline documentation/modules/ROOT/partials/modules/tutorial/proc-starting-mysql-database.adoc:37]: If you use Podman, run the following command:

File: documentation/modules/ROOT/partials/modules/tutorial/proc-starting-zookeeper.adoc [hunk 7]
```diff
@@ -28,6 +28,18 @@ which means the terminal's standard input and output are attached to the contain
 This enables other containers (and applications outside of the container) to communicate with ZooKeeper.
 --
 
+ifdef::community[]
+[NOTE]
+====
+For Podman use the below commands
```
Comment (roldanbob) [inline documentation/modules/ROOT/partials/modules/tutorial/proc-starting-zookeeper.adoc:34]: If you use Podman, run the following commands:

File: documentation/modules/ROOT/partials/modules/tutorial/proc-viewing-create-event.adoc [hunk 8]
```diff
@@ -29,6 +29,15 @@ Without this option, `watch-topic` would only show the events recorded after you
 Specifies that the output should include the event's key.
 In this case, this contains the row's primary key.
 
+[NOTE]
+====
+For Podman use the below command
```
Comment (roldanbob) [inline documentation/modules/ROOT/partials/modules/tutorial/proc-viewing-create-event.adoc:34]: If you use Podman, run the following command:

**Comment by rareddy** [comment]
@roldanbob The question also is,  do you want to adopt similarly for prod docs? thus removing the `ifdef` tags?

**Comment by roldanbob** [comment]
@rareddy Ah, yes, I think so, right? The podman instructions should work on OpenShift, and from what I understand, that is the preferred command set to use from the OpenShift perspective. Can you confirm that it works in that context?

**Comment by rareddy** [comment]
@roldanbob The podman instructions are not for Openshift, these are basically to run off the OpenShift. On Openshift there is only that one single build we would need to use podman to build connect based image.

**Comment by roldanbob** [comment]
@rareddy Yes, I think that we're saying the same thing? For the Debezium product, if we assume AMQ Streams on OpenShift, shouldn't we be leaning toward using podman? 
My understanding is that when we run on OpenShift, in all cases where users can run podman commands vs docker commands, we should provide the podman commands -- even if we also provide the equivalent docker commands.

**Comment by rareddy** [comment]
@roldanbob IMO, yes. But that is a generic question across all the documents rather than this particular PR. This PR (or the example itself) is not part of product so it should not apply here.

**Comment by rareddy** [comment]
@jpechane suggested changes are now in.

**Comment by roldanbob** [comment]
Thanks for the clarification @rareddy. I didn't realize that this PR was for something that's not exposed downstream.

**Comment by jpechane** [comment]
Applied, thanks guys!

**Pull request merged**